core/src/test/java/io/atomix/core/TestTest.java

```diff
@@ -0,0 +1,7 @@
+package io.atomix.core;
+
+/**
+ *
+ */
+public class TestTest {
```

Comment:
o yes!

Reply:
Ugh

Reply:
I'm actually not entirely sure how this got here. I did have a `TestTest` class, but it was a copy of the test from the related issue 🤷‍♂️